internal/controllers/addon/status_reporting.go

```diff
@@ -41,33 +39,16 @@ func (r *AddonReconciler) handleOCMAddOnStatusReporting(
 			setLastReportedStatus(addon)
 		}
 	}()
-	currentOCMAddonStatus, err := r.getAddonStatus(ctx, addon.Name)
+	//We've added upsert functionality in the status API so modifying ADO to use POST
+	log.Info("Making the Upsert call ")
+	err = r.postAddonStatus(ctx, addon)
```

Comment:
Mind linking the ticket / MR which accomplished the upsert-nature of POST addon-status endpoint?

Reply:
@ashishmax31 Do you have the ticket reference ?